File: core/src/main/java/com/victorlh/android/framework/core/dialog/IDialog.java
package com.victorlh.android.framework.core.dialog;

/**
 * Created by victor on 28/05/2017.
 */

public interface IDialog {

	void show();

	void close();

	void setOnCancelListener(EventosDialogs.OnCancelListener evento);

	void setOnDissmissListener(EventosDialogs.OnDissmissListener evento);
}

File: core/src/main/java/com/victorlh/android/framework/core/dialog/IDialog.kt
package com.victorlh.android.framework.core.dialog

/**
 * Created by victor on 28/05/2017.
 */
interface IDialog {
    fun show()
    fun close()
    fun setOnCancelListener(evento: (dialog: IDialog) -> Unit)
    fun setOnDissmissListener(evento: () -> Unit)
}